.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 lib
 es
+/dist/
```

Comment:
I prefer not having this in `.gitignore` - its often nice to look on the visual diff in the umd bundles, especially when updating deps etc

Reply:
it would also be helpful now to see what bundle got produced by this PR